cindy/index.js

```diff
@@ -0,0 +1,41 @@
+//32-PB-function-scope
+//Function Scope
+//1. Print Exponential Values
+//Write a function that accepts two numbers and validate that they are numbers.
+//After that, the function should print y exponential values starting from x.
+//For example if we have function(3, 5) The function should print 3 9 27 81 243. Prints 5 exponential values of 3.
+//function(2, 8) The function prints 2 4 8 16 32 64 128 256. Prints 8 exponential values of 2.
+const power = (base, exponent) =>{
+    
```

Comment:
Task 1: First of all you have to check if the numbers are numbers! after that you have to use for loop to print the numbers(2,4,8,16), means in line 9 you have to declare a brand new variable which is an empty string (let values="") and after line 12 you have to add  result into an empty string (values += `${result}`)
and at the end after line 13 do a console.log and you will see the numbers 2 4 8 .... 
console.log(values)

Reply:
done !  (but I had to search for it :c )